pcap_ex.c

```diff
@@ -290,7 +290,7 @@ pcap_ex_compile_nopcap(int snaplen, int dlt, struct bpf_program *fp, char *str,
 	char ebuf[PCAP_ERRBUF_SIZE];
 	int ret = -1;
 	
-	_mktemp(path);
+	mktemp(path);
```

Comment:
Yes, Windows compiler complained about it. I mistakenly "fixed" it earlier, but reverted back after verification in macOS and GNU/Linux.